pytest.ini

```diff
@@ -1,13 +1,12 @@
 [pytest]
 addopts =
     # show summary of all tests that did not pass
-    -ra
+    -rf
```

Comment:
i think this would actually have to happen in `common_utils.py`, specifically get rid of the `s`

also note that skip reasons would no longer be printed

Reply:
ok this is dumb, but if you change `-rfEsX` in `common_utils.py` to `-rsfEX` it might show the skips before the fails -> the fails are closer to the bottom -> scroll less, but you'd still need to take the test name and control f for the stacktrace

i have a change in test-infra for the hud log viewer to go to the stacktraces but when the hud log viewer is minimized its going to look like its just pointing at a line that says `==== FAILURES =====` so youd have to expand to see the name of the failing test and its stacktrace

Reply:
Is there a reason we want to show skips and xfails? It doesn't seem like they need to be in the short summary.

Reply:
skips because thats the only place skip reasons are printed, the X is for xpassed

Reply:
are we using the skip reasons for anything though? 🤔

Reply:
OK so I double checked and when I invoke pytest locally, it is only pytest.ini that is respected. So if we can't agree on how CI should work, I would still like to get the pytest.ini portion of this change in.